plugin/src/main/java/org/jenkinsci/plugins/workflow/cps/CpsFlowExecution.java

```diff
@@ -632,14 +634,74 @@ private CpsScript parseScript() throws IOException {
             trusted = new CpsGroovyShellFactory(this).forTrusted().build();
             shell = new CpsGroovyShellFactory(this).withParent(trusted).build();
 
-            s = (CpsScript) shell.reparse("WorkflowScript",script);
+            s = (CpsScript) shell.reparse("WorkflowScript", script);
 
             for (Entry<String, String> e : loadedScripts.entrySet()) {
                 shell.reparse(e.getKey(), e.getValue());
             }
-        } catch (RuntimeException | Error x) {
+        } catch (Exception x) {
```

Comment:
This loses handling of `Error`, especially `LinkageError`.

Reply:
I think I had some troubles compiling `catch (Exception | Error x)`, or just IDE warnings, can try.

Reply:
I see, it was an IDEA misconfiguration that had too old a "Language level" set to support multi-catches.

Reply:
Code updated to restore these `Throwable` descendant types.